src/features/auth/AuthContext.tsx

```diff
@@ -34,7 +34,7 @@ export function useAuthContext(): IAuthContext {
   return useContext(AuthContext)
 }
 
-export const AuthWrapper = ({ children }: { children: Element }) => {
+export const AuthWrapper = ({ children }: { children: JSX.Element }) => {
```

Comment:
```suggestion
export const AuthWrapper: FunctionComponent = ({ children }) => {
```

The issue could be that I believe the type is not `JSX.Element` but instead `React.ReactNode`.  Either way, I would suggest using the type generic `FunctionComponent` which will define the type of `children` for you!